### ISSUE TITLE
Fuzzy Skin does not work with reduced speed on overhangs

### ISSUE THREAD

**Issue by fridolin-parax** [bambulab/BambuStudio#4751]
### Bambu Studio Version

1.9.5.51

### Where is the application from?

Bambu Lab Official website

### OS version

Windows 10

### Additional system information

ryzen 7

### Printer

Bambu X1 

### How to reproduce

Load a Model with overhang, select the option use reduced speed for overhang and use the option fuzzy skin. When fuzzy skin is activated and you check the speed in the sliced preview you'll see that there is no speed reduction in the areas of overhang. It seems the fuzzy skin option is overwriting the speed setting of reduced speed.
Without fuzzy skin:
<img width="1403" alt="image" src="https://github.com/user-attachments/assets/9f46f64c-8813-4bfc-bd70-24fbe9788032">
With fuzzy skin:
![image](https://github.com/user-attachments/assets/99af525a-43a9-4b17-b3d8-9ee10b69ffd5)

### Actual results

Overhangs are printed really badly with fuzzy skin

### Expected results

Overhangs should print just as good as without fuzzy skin. 
This bug makes it impossible to produce properly with fuzzy skin (which is mainly usefull for production). 
A fast fix would be really appritiated.

### Project file & Debug log uploads

[Test for fuzzy Skin + Reduced speed.zip](https://github.com/user-attachments/files/16851972/Test.for.fuzzy.Skin.%2B.Reduced.speed.zip)

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by QingZhangBambu** [comment]
yes. fuzzy skin does not support overhang detection. i will mark this as a feature.

**Comment by fridolin-parax** [comment]
Thanks!
Do you now how long this will take?

Best

Fridolin

_______________________________________________________________
Fridolin Förster
CTO / Technischer Leiter
*#gerneperdu*

PARAX GmbH
Bockeldamm 19
59199 Bönen
D – Germany

Kontakt:
Mobil:      +49 (0) 163 4262 787
E-Mail:     ***@***.***
Web:        *www.parax.de* <www.parax.de>


Am Sa., 12. Okt. 2024 um 05:27 Uhr schrieb Shikieiki <
***@***.***>:

**Comment by QingZhangBambu** [comment]
I am not sure, concerned staff will review the feature. may take a long time

**Comment by M-A-B007** [comment]
+1
It's still an annoying issue in version 2.0.3.54.
Is there any workaround to reduce the speed in the specific areas??
